CLI.md

```diff
@@ -232,6 +232,9 @@ Open (i.e. activate) a new multi-owner chain deriving the UID from an existing o
 * `--fallback-duration-ms <FALLBACK_DURATION>` — The age of an incoming tracked or protected message after which the validators start transitioning the chain to fallback mode, in milliseconds
 
   Default value: `86400000`
+* `--execute-operations <EXECUTE_OPERATIONS>` — If this is `None`, all system operations and application operations are allowed. If it is `Some`, only operations from the specified applications are allowed, and no system operations
```

Comment:
I'm not sure if `None` and `Some` make sense in the context of CLI arguments?

Reply:
Good catch, thanks! Fixed now.